Tools/make-pkgconfig.swift

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env swift
+
+import Foundation
+
+/// An error indicating that the compiler's environment is not properly configured.
+struct EnvironmentError: Error, CustomStringConvertible {
+
+  /// The error's message.
+  let description: String
+
+  /// Creates an instance with given `messsage`.
+  init(_ messsage: String) { self.description = messsage }
+
+}
+
+extension String {
+
+  /// Returns `self` in which occurrences of new lines have been replaced by spaces.
+  func replacingNewlinesBySpaces() -> String {
```

Comment:
Nit: consider [`replaceingNewlinesWithSpaces`](https://www.quora.com/Whats-the-difference-between-replace-with-and-replace-by)